lib/vagrant-service-manager/command.rb

```diff
@@ -266,21 +265,17 @@ def print_vagrant_box_version(script_readable = false)
       end
 
       def restart_service(service)
-
-        if SCCLI_SERVICES.include? service
-          # TODO : Handle the case where user wants to pass extra arguments to OpenShift service
-          command = "sudo sccli #{service}"
-        else
-          command = "sudo systemctl restart #{service}"
-        end
-
-        with_target_vms(nil, { single_target: true}) do |machine|
-          machine.communicate.execute(command) do |type, error|
-            if type == :stderr
-              @env.ui.error(error)
-              exit 126
-            end
+        errors = []
+        command = "systemctl restart #{service}"
```

Comment:
@navidshaikh no it will not break restart because once your service is setup then user can use `systemctl` to do restart. It is one time setting which `sccli` do nothing else. One check you folks can add that if a user doing `restart` to not active service then instead trying to do systemctl restart put a warn message to user '{service} is not running' please run service first before restarting.